apps/TvDemoApp/Podfile

```diff
@@ -7,3 +9,16 @@ target 'TvDemoApp' do
   pod 'GoogleAds-IMA-tvOS-SDK', '~> 4.2'
   pod 'Mux-Stats-Google-IMA', '~> 0.9'
 end
+
+post_install do |installer|
+  installer.generated_projects.each do |project|
+    project.targets.each do |target|
+      target.build_configurations.each do |config|
+        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.0'
```

Comment:
question(non-blocking): Trying to grok this... assuming this is v12 for tvOS only? Can we use for iOS as well? If yes, would that reduce complexity here?

Reply:
Indeed, this should be `TVOS_DEPLOYMENT_TARGET`

Reply:
50825e2b